core/src/models/Export.ts

```diff
@@ -8,7 +8,10 @@ import {
   ForeignKey,
   DataType,
   Default,
+  AfterUpdate,
 } from "sequelize-typescript";
+import { appendFile } from "fs/promises";
```

Comment:
We still support node.js v12... so we can't use the fs/promises library yet :/

Reply:
Switched to fs-extra, but I believe it's just calling fs under the hood... According to this https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fspromises_appendfile_path_data_options it's been around since node 10?

Reply:
I think I disagree.

```
➜  ~ node --version
v12.22.8
➜  ~ node
Welcome to Node.js v12.22.8.
Type ".help" for more information.
> const fsp = require("fs/promises")
Uncaught Error: Cannot find module 'fs/promises'
Require stack:
- <repl>
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '<repl>' ]
}
```

Maybe it was hidden behind an experimental flag in v12.  Explaining that users seems... challenging.

Reply:
Aha! Now I see what's going on- fs/promises is available in v12, but it has to be imported like `require("fs").promises`.

```
Welcome to Node.js v12.22.8.
Type ".help" for more information.
> const fs = require("fs").promises
undefined
> fs
{
  access: [AsyncFunction: access],
  copyFile: [AsyncFunction: copyFile],
  open: [AsyncFunction: open],
  opendir: [Function: opendir],
  rename: [AsyncFunction: rename],
  truncate: [AsyncFunction: truncate],
  rmdir: [AsyncFunction: rmdir],
  mkdir: [AsyncFunction: mkdir],
  readdir: [AsyncFunction: readdir],
  readlink: [AsyncFunction: readlink],
  symlink: [AsyncFunction: symlink],
  lstat: [AsyncFunction: lstat],
  stat: [AsyncFunction: stat],
  link: [AsyncFunction: link],
  unlink: [AsyncFunction: unlink],
  chmod: [AsyncFunction: chmod],
  lchmod: [AsyncFunction: lchmod],
  lchown: [AsyncFunction: lchown],
  chown: [AsyncFunction: chown],
  utimes: [AsyncFunction: utimes],
  lutimes: [AsyncFunction: lutimes],
  realpath: [AsyncFunction: realpath],
  mkdtemp: [AsyncFunction: mkdtemp],
  writeFile: [AsyncFunction: writeFile],
  appendFile: [AsyncFunction: appendFile],
  readFile: [AsyncFunction: readFile]
}
> fs.appendFile
[AsyncFunction: appendFile]
> 
```

But when TS builds the code, it produces `require("fs/promises")`, which wasn't added until node 14. Using fs-extra solves the issue.